helm/idol-qms/values.yaml

```diff
@@ -104,17 +106,14 @@ single-agentstore:
     # -- the ingress controller path to access the agentstore index service with
     indexPath: "/agentstore-index/"
 
-  # -- the agentstore engine's query service name
-  queryserviceName: *agentStoreName
-
   # -- the agentstore engine's query service ACI port
   queryserviceACIPort: *agentstoreACIPort
 
   # -- the agentstore engine's index service name
   indexserviceName: "idol-agentstore-index-service"
```

Comment:
Suggestion was that we don't need the `idol-agentstore-index-service` for anything

Reply:
https://github.com/opentext-idol/idol-containers-toolkit/pull/27/commits/2b002fd3a32531ec3a7e8f5dd1413622514aea93